Library/Homebrew/utils/shell_env.sh

```diff
@@ -0,0 +1,71 @@
+keepkeys=(
```

Comment:
Favour capitalised variable names and run `shellcheck` on this file and address any complaints.